tests/unit/dependency/test_params.py

```diff
@@ -222,7 +222,9 @@ def test_get_hash_missing_param(tmp_dir, dvc):
 
 @pytest.mark.parametrize("param_value", ["", "false", "[]", "{}", "null"])
 def test_params_with_false_values(tmp_dir, dvc, param_value):
-    """These falsy params values should not ignored by `status` on loading."""
+    """
+    These false params values should not be ignored by `status` on loading.
```

Comment:
This is correct though, they are a `falsy` values.

Reply:
I guess codespell dictionary is wrong. We can skip this update and report this issue.

Reply:
Or, you can just add to the ignore-words-list:
https://github.com/iterative/dvc/blob/122a68398e17f21da1938129aa8cdb21123cb794/.pre-commit-config.yaml#L18-L19
.

Reply:
Related: https://github.com/codespell-project/codespell/issues/2444.